templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <style>
+      #{{ snippet_id }}-ffos-circling-fox {
+          /* All CSS should be preceeded by the ID selector for this individual
+            snippet. */
+      }
+    </style>
+    
+    {% if icon_url %}<a href="#" id="hello-cta">{% endif %}
```

Comment:
Use a class instead of an ID here. As per the [snippet development requirements](https://abouthome-snippets-service.readthedocs.org/en/latest/developing.html#snippet-requirements), you should avoid IDs that don't contain the snippet ID, and in general avoid them outside of the root div anyway.